pkg/scheduler/framework/plugins/imagelocality/image_locality.go

```diff
@@ -49,6 +52,24 @@ func (pl *ImageLocality) Name() string {
 	return Name
 }
 
+// Filtering and scoring based on container image names.
+func (pl *ImageLocality) SignPod(ctx context.Context, pod *v1.Pod) ([]fwk.SignFragment, *fwk.Status) {
+	nameSet := sets.New[string]()
+
+	containers := []v1.Container{}
+	containers = append(containers, pod.Spec.Containers...)
+	containers = append(containers, pod.Spec.InitContainers...)
+
+	for _, container := range containers {
+		nameSet.Insert(normalizedImageName(container.Image))
+	}
+	names := nameSet.UnsortedList()
+	sort.Slice(names, func(i, j int) bool {
+		return names[i] < names[j]
+	})
```

Comment:
You can use `List` method, it returns a sorted slice:
```suggestion
	names := nameSet.List()
```

Reply:
Nice!